content/docs/codebase-overview.md

```diff
@@ -104,9 +104,9 @@ if (__DEV__) {
 
 ### Flow {#flow}
 
-We recently started introducing [Flow](https://flow.org/) checks to the codebase. Files marked with the `@flow` annotation in the license header comment are being typechecked.
+Recientemente se introdujo validaciones [Flow](https://flow.org/) al código base. Archivos marcados con la anotación `@flow` en el comentario de encabezado de la licencia se están validando.
 
-We accept pull requests [adding Flow annotations to existing code](https://github.com/facebook/react/pull/7600/files). Flow annotations look like this:
+Se aceptan PR's [para agregar anotaciones Flow al código existente](https://github.com/facebook/react/pull/7600/files). Las anotaciones Flow son así:
```

Comment:
I suggest `Aceptamos pull requests...` to make this clearer.

Reply:
```suggestion
Aceptamos pull requests [para agregar anotaciones Flow al código existente](https://github.com/facebook/react/pull/7600/files). Las anotaciones Flow son así:
```

Reply:
I think this one is duplicated. Already done by the way!